reStream.sh

```diff
@@ -170,7 +176,9 @@ case "$rm_version" in
                 echo "Using the newer :mem: video settings."
                 bytes_per_pixel=2
                 pixel_format="gray16be"
-                video_filters="$video_filters colorlevels=rimin=0:rimax=29/255:gimin=0:gimax=29/255:bimin=0:bimax=29/255,transpose=3"
```

Comment:
@BWindey Can you remove these comments? Or replace them with an explanation what the filter actually does and why it is needed.

@itay-grudev you initially implemented this `colorlevels` filter, do you see any issues with switching to the `curves` filter?

I do not have a remarkable2 so I cannot compare which one of the filters gives the best result (visually and performance wise).

Reply:
With curves:
![image](https://github.com/user-attachments/assets/8a0a7c1d-61fc-4dad-9a99-4ee82bac49e7)
With previous method:
![image](https://github.com/user-attachments/assets/68970c41-7635-4a1b-b1d0-d296f99d2ebe)


To me, they are identical. At least with the black and white content.
On an image with 2 different "marker", both methods show the same 2 different grey tints.

I tried to look at performance by looking at `top`, but there didn't seem to be any difference. The filter happens on the computer anyway.


As comment I would propose this:
```sh
# "curves" modifies the intensity of an RGB-value.
# The remarkable's max output is 1/8th of the normal brightness
# so the curves filter sets the scale from 0 -> 1 to 0 -> 0.125
# (https://ffmpeg.org/ffmpeg-filters.html#toc-Examples-63)
video_filters="$video_filters,curves=all='0/0 0.125/1 1/1',transpose=3"
```
What do you think?
The previous method did not have any comments.

Reply:
The comment sounds good. I don't see a visual difference either.

If you update the PR with the comment I will merge.